src/main/java/dk/kb/elivagar/pubhub/PubhubPacker.java

```diff
@@ -169,28 +168,25 @@ public void packFileForEbook(File bookFile) throws IOException {
         log.info("Packaging book file for book-id: " + id);
         File bookDir = getBookDir(id, BookTypeEnum.EBOG);
         File bookLinkFile = new File(bookDir, bookFile.getName());
-        if(bookLinkFile.isFile()) {
-            log.trace("The hard link for the book file for book-id '" + id + "' already exists.");
-        } else {
-            Files.createLink(bookLinkFile.toPath(), bookFile.toPath().toAbsolutePath());
-        }
+        //  The file is overwritten if it already exists
+        FileUtils.copyFileFollowSymbolicLinks(bookFile, bookLinkFile);
```

Comment:
Should the name of the copyFileFollowSymbolicLinks method be changed?

Reply:
Renamed to copyFile